cmake/templates/cvconfig.h.in

```diff
@@ -100,6 +100,9 @@
 #cmakedefine HAVE_OPENCL_STATIC
 #cmakedefine HAVE_OPENCL_SVM
 
+/* NVIDIA OpenCL Extensions support */
+#cmakedefine HAVE_CLNVEXT
```

Comment:
Please use more readable name:
```
HAVE_OPENCL_NV_EXT
```

or more concrete:
```
HAVE_OPENCL_D3D11_NV
```

Reply:
changed to HAVE_OPENCL_D3D11_NV